src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -95,7 +95,10 @@ const Alert = styled(
           {typeof icon === 'string' ? <InlineSvg src={icon} size={iconSize!} /> : icon}
         </IconWrapper>
       )}
-      <StyledTextBlock>{children}</StyledTextBlock>
+      <StyledTextBlock>
+        {/* wrap with <span> to preserve inline white space in children */}
+        <span>{children}</span>
```

Comment:
@rhcarvalho This is going to make it hard if we need to customize the contents of the Alert.

Reply:
The alternative was fixing each individual alert box.

This is one of the boxes we had in prod for a couple of days:

![image](https://user-images.githubusercontent.com/88819/77737053-80710e00-700d-11ea-85a2-48ba48531d67.png)

My intention here was just to bring back the correct look of alerts (and I did check all of the ones we capture in Percy).

If this is really a problem, we need to revisit how we are aligning the icon, see #17840 where we introduced the bug when we changed the box model to `flex`. Most likely we don't need to make `StyledTextBlock` be `display: flex`.